woocommerce/woocommerce-android
# Add Screen to updated product when it's not stock-managed

Closes: #10363 


### Description


This PR replaces the snackbar displayed when the product is not stock-managed with a new bottom sheet that allows the user to either go to the product details screen or to change the option to manage stock for that product and then update the quantity. 

This is the screen:

<img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/1a72a65c-e9da-496b-b005-4b6c9ccb9609" width=250>

Designs: w3JBReW1UJesCf02sj43kR-fi

### Testing instructions


- Go to products list
- click on the scan icon
- scan a product that is not stock managed
- make sure the stock management bottom sheet is displayed
- test the view product details button
- test the Manage stock option and make sure it takes you to the Quantity update bottom sheet
- make sure you are able to increase the product quantity

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/30724184/d09f2802-4bfa-48ac-a48f-c6abbf899d9f


https://github.com/woocommerce/woocommerce-android/assets/30724184/a8aa8240-4bb9-4061-87dd-76967308eced
